packages/deployer/test/sample-project/contracts/mixins/OwnerMixin.sol

```diff
@@ -7,6 +7,11 @@ contract OwnerMixin is OwnerNamespace {
     /* MODIFIERS */
 
     modifier onlyOwner() {
+        require(msg.sender == _ownerStorage().owner, "Only owner allowed");
```

Comment:
Another fly by improvement. Before, every onlyOwner check would check if `owner == address(0)`. Now it only does it on the first time.